src/main/java/edu/oregonstate/cs361/battleship/Ship.java

```diff
@@ -2,23 +2,22 @@
 
 import static edu.oregonstate.cs361.battleship.BattleshipModel.GRID_SIZE;
 
-class Ship {
+public class Ship {
```

Comment:
Just form, and ocd. Without a modifier, it's not "world" accessible. Public adds that accessibility.

Although... it doesn't make a difference in this case....